api/src/main/java/ai/djl/nn/Activation.java

```diff
@@ -258,6 +258,29 @@ public static NDList selu(NDList arrays) {
         return new NDList(arrays.singletonOrThrow().getNDArrayInternal().selu());
     }
 
+    /**
+     * Applies tabNetGLU activation(which is mostly used in tabNet) on the input {@link NDArray}.
+     *
+     * @param array the input {@link NDArray}
+     * @param units the half number of the resultant features
+     * @return the {@link NDArray} after applying tabNetGLU function
+     */
+    public static NDArray tabNetGLU(NDArray array, int units) {
```

Comment:
If it is (mostly) specialized to tabnet, then it is probably better to put the activation on the Tabnet class instead and leave the `Activation` class for general purpose activations. As long as you make it `public static`, it can still be accessed just as easily.